modules/EnsEMBL/Web/Component/Variation/Summary.pm

```diff
@@ -50,7 +50,8 @@ sub content {
     $self->object->vari_class eq 'SNP' ? () : $self->three_prime_co_located(),
     $self->synonyms,
 ## EG
-    $self->inter_homoeologues,
+    $self->primer_marker,
+    #$self->inter_homoeologues,

```

Comment:
Yes, this generic method is replaced by the new "primer_marker" method.
However, because I didn't write `inter_homoeologues`, I didn't want to remove it.

Reply:
ok, thanks